packages/common/src/discussions/api/types.ts

```diff
@@ -799,15 +789,7 @@ export interface IChannelMetadata {
  * @interface ICreateChannelPermissions
  */
 export interface ICreateChannelPermissions {
-  access?: SharingAccess;
-  allowAnonymous?: boolean;
-  groups?: string[];
-  orgs?: string[];
-  /**
-   * Not available until the V2 Api is released
-   * @hidden
-   */
-  channelAclDefinition?: IChannelAclPermissionDefinition[];
+  channelAclDefinition: IChannelAclPermissionDefinition[];
```

Comment:
This will have impact on the Discussion Board workspace UI where we create channels. We'll now need to build ACL rules there vs selecting groups/access/allowAnonymous